### PR TITLE
Contribute: Testing ZAP

### PR DESCRIPTION
**Pull request by psiinon**
I think this is worth making into a separate 'task' as it might appeal to people without any/enough coding skills...

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: site/content/docs/contribute/testing.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Contribute: Test ZAP"
+type: page
+---
+
+ZAP is designed to attack real world apps, so we do not tune ZAP just to score better against vulnerable test apps.
+However such test apps are still very worthwhile and can be representative of real world apps.
```
Comment (kingthorin) [inline site/content/docs/contribute/testing.md:7]:
```suggestion
However, such test apps are still very worthwhile and can be representative of real world apps.
```

File: site/content/docs/contribute/testing.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+There are a large number of deliberately vulnerable apps - see the 
+[OWASP VWAD](https://owasp.org/www-project-vulnerable-web-applications-directory/) project.
+
+We would love to test ZAP against all of them, but these things take time and ZAP has a relatively small 
+[Core Team](/docs/team/) so we cannot do everything that we want to.
+
+This is another area where you can help!
+
+Try using ZAP to test one of the apps that we are not [testing ZAP regularly against](/docs/scans/).
```
Comment (kingthorin) [inline site/content/docs/contribute/testing.md:17]:
```suggestion
Try using ZAP to test one of the apps that we are not [regularly testing ZAP against](/docs/scans/).
```

**Comment by psiinon** [comment]
Thanks - applied those suggestions 😄

**Review by thc202 — APPROVED** [review]

**Comment by psiinon** [comment]
Updated to link to the relevant FAQs

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Pull request merged**